src/store/state/state-model.ts

```diff
@@ -113,10 +115,12 @@ export const stateModel: StateModelIF = {
     folioNumber: '',
     isPriority: false
   },
+  staffPaymentValidity: false,
```

Comment:
Please consider combining this and the staffPayment object above into a step object (same as the others).

Reply:
Done

Reply:
Way better - thanks!